src/toil/cwl/cwltoil.py

```diff
@@ -2139,14 +2162,21 @@ def main(args: Union[List[str]] = None, stdout: TextIO = sys.stdout) -> int:
                 logging.error(err)
                 return 33
             runtime_context.secret_store = SecretStore()
-            initialized_job_order = cwltool.main.init_job_order(
-                job_order_object,
-                options,
-                tool,
-                jobloader,
-                sys.stdout,
-                secret_store=runtime_context.secret_store,
-            )
+
+            try:
+                initialized_job_order = cwltool.main.init_job_order(
+                    job_order_object,
+                    options,
+                    tool,
+                    jobloader,
+                    sys.stdout,
+                    secret_store=runtime_context.secret_store,
+                )
+            except SystemExit as e:
+                if e.code == 2:  # raised by argparse's parse_args() function
+                    print('If both a cwl file and a yml file were provided, this may be the argument order.' +
```

Comment:
```suggestion
                    print('If both a CWL file and an input object (YAML/JSON) file were provided, this may be the argument order.' +
```